include/bbp/sonata/config.h

```diff
@@ -225,6 +225,27 @@ class SONATA_API SimulationConfig
         /// The sorting order of the spike report. Default is "by_time"
         SpikesSortOrder sortOrder;
     };
+
+    struct ModificationBase {
+        enum class ModificationType { invalid = -1, TTX, ConfigureAllSections };
+
+        /// Node set which receives the manipulation
+        std::string nodeSet;
+        /// Name of the manipulation. Supported values are “TTX” and “ConfigureAllSections”.
+        ModificationType type;
+    };
+
```

Comment:
I wonder if `ModifcationBase` shouldn't be exposed in the API, but a `struct ModificationTTX: public ModificationBase {}` could be exposed instead?

Reply:
Good point. I've added`ModificationTTX`.